draft-ietf-quic-http.md

```diff
@@ -1340,7 +1340,9 @@ setting is the default value. When a 0-RTT QUIC connection is being used, the
 initial value of each server setting is the value used in the previous session.
 Clients MUST store the settings the server provided in the session being resumed
 and MUST comply with stored settings until the current server settings are
-received.
+received.  A client uses these initial values in a situation where it wants to
```

Comment:
```suggestion
received.  A client uses these initial values in situations where it wants to
```

Reply:
How about

> A client uses these initial values when it wants to send requests...

Or perhaps remove the "want" altogether and simplify it further to

> A client uses these initial values to send requests...